src/index.ts

```diff
@@ -71,6 +71,11 @@ export interface TypeInfo {
   comment: string
 }
 
+let envTransformer
+if (process.env['TS_NODE_TRANSFORMERS']) {
+  envTransformer = require(join(process.cwd(), process.env['TS_NODE_TRANSFORMERS'] as string))
```

Comment:
How would someone specify more than one here? Also, this logic is duplicated from `cli` - could we not just support strings from the options input?